docs/descriptor.rst

```diff
@@ -0,0 +1,210 @@
+Image descriptor file
+=====================
+
+Most important, user facing part of Concreate is the descriptor file. We use
+`YAML <http://yaml.org/>`_ format which is easy to read and edit by humans but still machine
+parseable.
+
+.. note::
+
+    By convention we use the ``image.yaml``  file name for the image descriptor.
+
+Below you can find description of image descriptor sections.
+
+General information
+-------------------
+
+.. glossary::
+
+    ``name`` (required)
+        Image name without the registry part.
+
+    ``version`` (required)
+        Version of the image.
+
+    ``from`` (required)
+        Base image of your image.
+
+    ``schema_version`` (required)
+        Here you specify the schema version of the descriptor. This influences what versions of Concreate are able to parse it.
+
+    ``description``
+         Short summary of the image -- also used to create the ``description`` label.
+
+    ``user``
+        The user that should be used to launch the main process in the container. Can be name or uid. See `OCI spec <https://github.com/opencontainers/image-spec/blob/master/config.md#properties>`_ for more information.
+
+.. code:: yaml
+
+    name: "jboss-eap-7/eap70-openshift"
+    version: "1.4"
+    from: "jboss-eap-7-tech-preview/eap70:1.2"
+    description: "Red Hat JBoss Enterprise Application 7.0 - An application platform for hosting your apps that provides an innovative modular, cloud-ready architecture, powerful management and automation, and world class developer productivity."
+    user: "alice"
+
+Labels
+------
+
+Every image can include labels. Concreate makes it easy to do so with the ``labels`` section.
+
+.. code:: yaml
+
+    labels:
```

Comment:
I would mention and link https://github.com/projectatomic/ContainerApplicationGenericLabels as a resource to study/see.

Reply:
Good idea.